generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -276,6 +270,13 @@ class FireSimSmallSystemConfig extends Config(
   new freechips.rocketchip.subsystem.WithInclusiveCache(nWays = 2, capacityKB = 64) ++
   new chipyard.RocketConfig)
 
+class FireSimDmiRocketConfig extends Config(
```

Comment:
Is the `WithNoSerialTL` the right thing here? I think we should tie it off in the harness instead of modifying the target.

Reply:
The problem is that we have both bridge harness binders auto-added to all designs if the IO's exist. This is a quick way to just disable one of the IOs so that the harness connects properly

Reply:
Just add a OverrideHarnessBinder that ties it off?